conf/distro/rpb-qt5.conf

```diff
@@ -0,0 +1,8 @@
+require conf/distro/include/rpb.inc
+
+DISTRO_NAME = "Reference-Platform-Build-QT5"
+
+PACKAGECONFIG_append_pn-qtbase = " gles2 eglfs fontconfig kms"
+
+DISTRO_FEATURES_remove = "wayland x11"
+DISTRO_FEATURES_append = " pam systemd opengl"
```

Comment:
this is taken care by rpb.inc already. you shouldn't need them here, and should rely on the defaults from rpb.inc